scanner/src/funTest/kotlin/storages/AbstractProvenanceBasedScanStorageFunTest.kt

```diff
@@ -61,22 +62,24 @@ abstract class AbstractProvenanceBasedScanStorageFunTest(vararg listeners: TestL
             "succeed for a valid scan result" {
                 val scanResult = createScanResult()
 
-                storage.write(scanResult)
+                val writeResult = storage.write(scanResult)
                 val readResult = storage.read(scanResult.provenance as KnownProvenance)
 
                 readResult should containExactly(scanResult)
+                writeResult shouldBe true
```

Comment:
Nit: Verify `writeResult` before `readResult` to match the execution order.
And you could make it one-liners (same for the other tests):
```
storage.write(scanResult) shouldBe true
storage.read(scanResult.provenance as KnownProvenance) should containExactly(scanResult)
```